ui_testing/testcases/basic_tests/test05_testplans.py

```diff
@@ -385,37 +386,60 @@ def test011_test_plan_completed_to_inprogress(self):
 
-        self.base_selenium.LOGGER.info(
-            'Attempting to create another testplan with the same data as the previously created one')
+        if ("same" == same):
+            self.base_selenium.LOGGER.info(
+                'Attempting to create another testplan with the same data as the previously created one')
 
-        # create another testplan with the same data
-        self.test_plan.create_new_test_plan(name=testplan_name, material_type=testplan['materialType'],
-                                            article=(testplan['article'])[0])
+            # create another testplan with the same data
+            self.test_plan.create_new_test_plan(name=testplan_name, material_type=testplan['materialType'],
+                                                article=(testplan['article'])[0])
 
-        self.base_selenium.LOGGER.info(
-            'Waiting for the error message to make sure that validation forbids the creation of two testplans having the same name, material type and article')
-        validation_result = self.base_selenium.wait_element(element='general:oh_snap_msg')
+            self.base_selenium.LOGGER.info(
+                'Waiting for the error message to make sure that validation forbids the creation of two testplans having the same name, material type and article')
+            validation_result = self.base_selenium.wait_element(element='general:oh_snap_msg')
 
-        self.base_selenium.LOGGER.info(
-            'Assert the error message to make sure that validation forbids the creation of two testplans having the same name, material type and article? {}'.format(
-                validation_result))
-        self.assertTrue(validation_result)
+            self.base_selenium.LOGGER.info(
+                'Assert the error message to make sure that validation forbids the creation of two testplans having the same name, material type and article? {}'.format(
+                    validation_result))
+            self.assertTrue(validation_result)
+
+        else:
+            self.base_selenium.LOGGER.info(
+                'Attempting to create another testplan with the same data but All Articles')
+
+            self.test_plan.create_new_test_plan(name=testplan_name, material_type=testplan['materialType'],
+                                                article='All')
+
+            self.base_selenium.LOGGER.info(
+                'Waiting for the error message to make sure that validation forbids the creation of two testplans having the same name, material type and one with ALL article')
+            validation_result = self.base_selenium.wait_element(element='general:oh_snap_msg')
+
+            self.base_selenium.LOGGER.info(
+                'Assert the error message to make sure that validation forbids the creation of two testplans having the same name, material type and one with All article? {}'.format(
+                    validation_result))
+            self.assertTrue(validation_result)
```

Comment:
I can see that the only difference between "if" and "else" code is the value of "article" in the `self.test_plan.create_new_test_plan` so, just do this 
`article = testplan['article'])[0] if same == "same" else "All"` then pass this `article` to the `create_new_test_plan` method

Reply:
done